Update Node Operator Spork/Epoch Documentation (#862)
onflow/flow: docs/content/node-operation/node-bootstrap.mdx

This command will detect and provide information about common misconfigurations, or confirm that the machine account is configured correctly.

## Step 3 - Start Your Flow Node

Ensure you have configured your node using the [Node Setup guide](/node-operation/node-setup).

### Confirming authorization

> You can confirm your node's successful registration and authorization by reading the [Staking Contract](/core-contracts/staking-contract-reference#contract).

Reviewer: ```suggestion
You can confirm your node's successful registration and authorization by executing a Cadence script to query the [Staking Contract](/core-contracts/staking-contract-reference#contract).
```

> At the end of the `EpochStaking` phase, the members of the Proposed Identity Table are confirmed as authorized participants in the next epoch. 

Reviewer: ```suggestion
At the end of the `STAKINGAUCTION` phase, the members of the Proposed Identity Table are confirmed as authorized participants in the next epoch. 
```
In the contract, it is referred to as staking auction, so I assume we want to keep calling it that here?

Author: I'll re-use the terminology from https://docs.onflow.org/staking/epoch-preparation/#phase-transitions:
* `Staking Auction Phase`
* `Epoch Setup Phase`
* `Epoch Committed Phase`

Author: Updated these in https://github.com/onflow/flow/pull/862/commits/29ba2f34aaf3d71d5badf5f8f5afac585deae3c1

> Therefore, if your node ID appears in the Proposed Identity Table during the `EpochStaking` phase, your node will be a participant in the next epoch.

Reviewer: ```suggestion
Therefore, if your node ID appears in the Proposed Identity Table during the `STAKINGAUCTION` phase, your node will be a participant in the next epoch.
```

You can read the current Proposed Identity Table using the [getProposedTable script](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/scripts/get_proposed_table.cdc).

> You can read the current epoch phase using the [getEpochPhase script](https://github.com/onflow/flow-core-contracts/blob/master/transactions/epoch/scripts/get_epoch_phase.cdc). (A return value of `0` indicates the `EpochStaking` phase.)

Reviewer: ```suggestion
You can read the current epoch phase using the [getEpochPhase script](https://github.com/onflow/flow-core-contracts/blob/master/transactions/epoch/scripts/get_epoch_phase.cdc). (A return value of `0` indicates the `STAKINGAUCTION` phase.)
```

### Trusted Root Snapshot

Once your node has been registered and authorized by the service account, it will be able to participate in the next epoch.

![Flow Epoch Schedule](https://storage.googleapis.com/flow-resources/documentation-assets/epoch-startup-order.png)

A new node must bootstrap with a trusted root snapshot of the protocol state, where the node is a confirmed participant. 
Since new nodes are confirmed at the end of the `EpochStaking` phase, this means that, if the node is registered to join at epoch `N+1`, it must use a root snapshot from within the `EpochSetup` phase of epoch `N`.

### Dynamic Startup

Flow provides a mechanism called Dynamic Startup to simplify the process of obtaining the root snapshot.

> When using Dynamic Startup, the node can be started **at any time during the `EpochStaking` phase**.

Reviewer: ```suggestion
When using Dynamic Startup, the node can be started **at any time during the `STAKINGAUCTION` phase**.
```
